ui/js/util/getThemes.js

```diff
@@ -0,0 +1,20 @@
+const { readdirSync } = require("fs");
```

Comment:
I'm pretty sure this can't go inside of a selector, so if we can't get it into package.json, the code for loading the themes ought to be in an action.